dashboard/test/fixtures/resource.yml

```diff
@@ -3,21 +3,25 @@ resource_101:
   key: resource_101
   name: Code Studio
   url: code.org
+  course_version: course_version_2018
 resource_102:
   id: 102
   key: resource_102
   name: wiki
   url: wikipedia.org
+  course_version: course_version_2018
 resource_103:
   id: 103
   key: resource_103
   name: class slides
   url: docs.google.com/slides
+  course_version: course_version_2018
 resource_104:
   id: 104
   key: resource_104
   name: class site
   url: testwebsite.fake
+  course_version: course_version_2018
```

Comment:
just confirming that you were able to drop / reload the test DB after making these changes?

Reply:
Yeah, I used `RAILS_ENV=test bundle exec rake db:reset db:test:prepare` but the reset might not have been necessary, especially if the migration will remove course-version-less resources.